src/dbos-executor.ts

```diff
@@ -112,6 +112,8 @@ export interface DBOSConfig {
 
   applicationVersion?: string;
   executorID?: string;
+
+  listenQueues?: WorkflowQueue[];
```

Comment:
Well... I asked in the other repo why can't we use listenQueues in DBOSConfig in Python but the answer was an explicit function was better. I'd love to see those interface more consistent across languages. Can we have a new `listenQueues` function instead?